2-mandatory/4-people-i-know.js

```diff
@@ -394,6 +405,14 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
+let friendsWho = friends.filter(function(person){
+  person.colleagues.find(function(e){
+    if(e.name === "Stacie Villarreal"){
+      friendsWhoAreColleaguesOfStacie.push(person.name.first +' '+ person.name.last)
+    }
+  })
+})
```

Comment:
This works but is a bit confusing because you aren't returning anything in `.find()` (and not using the response from it). Have a look at the docs for it here and what the return type is: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

I would suggest either changing it to a `forEach` or trying to adjust it to use `.find()` according to the docs.

Well done for trying `find()` though!